airbyte-integrations/connectors/source-salesforce/source_salesforce/rate_limiting.py

```diff
@@ -34,7 +34,7 @@ def should_give_up(exc):
         if exc.response is not None and exc.response.status_code == codes.forbidden:
             error_data = exc.response.json()[0]
             if error_data.get("errorCode", "") == "REQUEST_LIMIT_EXCEEDED":
-                give_up = False
+                give_up = True
```

Comment:
Custom backoff handler is used in call calls BULK API. if 403 (Rate Limit) is received no need in sleeping.